src/target/target_flash.c

```diff
@@ -0,0 +1,271 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2022  Black Sphere Technologies Ltd.
```

Comment:
Please use ` * Copyright (C) 2022 1BitSquared <info@1bitsquared.com>` in the attribution line here.

Reply:
ooops, thats a pretty bad copy pasta